composer.json

```diff
@@ -45,7 +45,7 @@
         "akeneo/phpspec-skip-example-extension": "^4.0",
         "doctrine/orm": "^2.5",
         "gedmo/doctrine-extensions": "^2.4",
-        "lakion/api-test-case": "^3.1.0",
+        "lchrusciel/api-test-case": "^3.1.0",
```

Comment:
Can we update it to '^4.1' as well?

Reply:
Sure, currently this fails due to some unknown error.